docs/paper/dev/api/event-api/event-listeners.md

```diff
@@ -106,12 +106,12 @@ Some events can be cancelled, preventing the given action from being completed.
 public class ExampleListener implements Listener {
 
     @EventHandler
-    public void onPlayerJoin(PlayerJoinEvent event) {
+    public void onAnvilDamagedEvent(AnvilDamagedEvent event) {
         event.setCancelled(true);
     }
 }
 ```
-
+**The BlockLockCheckEvent method and the PlayerLoginEvent method do not implement the Cancellable interface, but similar effects can still be achieved through the setResult (Result result) method.**
```

Comment:
```suggestion
**The BlockLockCheckEvent and PlayerLoginEvent types do not implement the Cancellable interface, but similar effects can still be achieved through the setResult (Result result) method.**
```

Reply:
I'm not sure such random misnomer notes should exist within the event listener docs, at least, not pointing to specific events; I'd much rather something like this exist but as a more generic nod towards the JDs

Reply:
Thank you very much. I think your statement is more reasonable.

Reply:
> 我不确定事件侦听器文档中是否应该存在这样随机的用词不当的注释，至少不指向特定事件；我宁愿存在这样的东西，但作为对 JD 的更通用的点头



> I'm not sure such random misnomer notes should exist within the event listener docs, at least, not pointing to specific events; I'd much rather something like this exist but as a more generic nod towards the JDs

I advocate this change because I only found that these two classes exist in this case. I think that if there is no special explanation, developers may not know how to implement similar functions (at least I do)

Reply:
> I'm not sure such random misnomer notes should exist within the event listener docs, at least, not pointing to specific events; I'd much rather something like this exist but as a more generic nod towards the JDs

I mean, we don't really have a type for "result based events" like the login event etc.
For all I care, we could write proper docs in the Cancellable interface or on the specific events.
Don't think that makes it invalid to at least point out that, next to the cancellation type, some events types also accept a result. Tho this is really not a hill for me to die on 😓

Reply:
> > I'm not sure such random misnomer notes should exist within the event listener docs, at least, not pointing to specific events; I'd much rather something like this exist but as a more generic nod towards the JDs
> 
> I mean, we don't really have a type for "result based events" like the login event etc. For all I care, we could write proper docs in the Cancellable interface or on the specific events. Don't think that makes it invalid to at least point out that, next to the cancellation type, some events types also accept a result. Tho this is really not a hill for me to die on 😓

I think I understand. I will make some changes.